### ISSUE TITLE
[Errno 30] Read-only file system: '/opt/python/lib/python3.8/site-packages/validate_email/data'

### ISSUE THREAD

**Issue by airmonitor** [karolyi/py3-validate-email#13]
Hi.

Sadly still with ver 0.2.1 I'm getting issue related to writting to /opt/python/lib/python3.8/site-packages/validate_email/data :/

I suppose that you didn't had time to move data file to TEMP directory yet?

```
[ERROR] OSError: [Errno 30] Read-only file system: '/opt/python/lib/python3.8/site-packages/validate_email/data'
Traceback (most recent call last):
  File "/var/task/serverless_sdk/__init__.py", line 130, in wrapped_handler
    return user_handler(event, context)
  File "/var/task/s_EmailNotification.py", line 23, in error_handler
    raise e
  File "/var/runtime/bootstrap.py", line 131, in handle_event_request
    response = request_handler(event, lambda_context)
  File "/var/task/serverless_sdk/__init__.py", line 130, in wrapped_handler
    return user_handler(event, context)
  File "/var/task/s_EmailNotification.py", line 23, in error_handler
    raise e
  File "/var/runtime/bootstrap.py", line 131, in handle_event_request
    response = request_handler(event, lambda_context)
  File "/var/task/serverless_sdk/__init__.py", line 130, in wrapped_handler
    return user_handler(event, context)
  File "/var/task/s_EmailNotification.py", line 23, in error_handler
    raise e
  File "/var/task/s_EmailNotification.py", line 18, in <module>
    user_handler = serverless_sdk.get_user_handler('email_notification.handler')
  File "/var/task/serverless_sdk/__init__.py", line 54, in get_user_handler
    user_module = import_module(user_module_name)
  File "/var/lang/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/task/email_notification.py", line 13, in <module>
    from validate_email import validate_email
  File "/opt/python/lib/python3.8/site-packages/validate_email/__init__.py", line 1, in <module>
    from .validate_email import validate_email, validate_email_or_fail  # NOQA
  File "/opt/python/lib/python3.8/site-packages/validate_email/validate_email.py", line 4, in <module>
    from .domainlist_check import domainlist_check
  File "/opt/python/lib/python3.8/site-packages/validate_email/domainlist_check.py", line 7, in <module>
    from .updater import (
  File "/opt/python/lib/python3.8/site-packages/validate_email/updater.py", line 24, in <module>
    LIB_PATH_DEFAULT.mkdir(exist_ok=True)
  File "/var/lang/lib/python3.8/pathlib.py", line 1279, in mkdir
    self._accessor.mkdir(self, mode)
```

regards
Tom

**Comment by karolyi** [comment]
Hey,

I made a mistake there, the data dir doesn't get added to the installed package. I have fixed the issue in 0.2.2. Please have a go with it.

**Comment by karolyi** [comment]
@airmonitor try with 0.2.5 please, and sorry for the inconvenience. Python package management is a mess. It should work now as expected.
